FAQ.md

```diff
@@ -35,7 +37,9 @@
     *   `getting-started`: issue or pull request is suitable for someone new to GitHub and/or this project
     *   `help-wanted`: a question or request for assistance
     *   `left-as-was`: marks an issue closed because the item in question will be left as it was
+    *   `merge`: issue or pull request related to merge changes from "third-party" repository
```

Comment:
This wording is a bit vague. (in theory, every pull request involves merging *something*!)

Suggested wording: "issue or pull request involving changes from an upstream repository, eg incorporation of updates from lesson-template into a specific lesson"